src/container/rpc.h

```diff
@@ -76,9 +76,6 @@
 	X(CONT_EPOCH_DISCARD,						\
 		0, &CQF_cont_epoch_op,					\
 		ds_cont_op_handler, NULL),				\
-	X(CONT_EPOCH_COMMIT,						\
```

Comment:
Why you removed this and changed the dc_tx_commit()? I'm not sure if it'll be used in later DAOS transaction APIs?